membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -39,8 +39,6 @@ class TouchpointComponents(stage: String)(implicit system: ActorSystem) {
   lazy val digitalPackConf = environmentConf.getConfig(s"zuora.ratePlanIds.digitalpack")
   lazy val paperCatalogConf = environmentConf.getConfig(s"zuora.productIds.subscriptions")
   lazy val membershipConf = environmentConf.getConfig(s"zuora.ratePlanIds.membership")
-  lazy val sfOrganisationId = environmentConf.getString("salesforce.organization-id")
```

Comment:
This used to be used here:
https://github.com/guardian/members-data-api/blob/7efcb6a08b49bb1080544520ba424c408458b733/membership-attribute-service/app/controllers/SalesforceHookController.scala#L46
(this controller has now been deleted)